magicctapipe/scripts/lst1_magic/semi_automatic_scripts/config_general.yaml

```diff
@@ -0,0 +1,40 @@
+mc_tel_ids:
+    LST-1: 1
+    LST-2: 0
+    LST-3: 0
+    LST-4: 0
+    MAGIC-I: 2
+    MAGIC-II: 3
+
+directories:
+    workspace_dir : "/fefs/aswg/workspace/elisa.visentin/auto_MCP_PR/"  # Output directory where all the data products will be saved.
+    # MC paths below are ignored if you set NSB_matching = true.
+    MC_gammas     : "/fefs/aswg/data/mc/DL0/LSTProd2/TestDataset/sim_telarray"  # set to "" if you don't want to process these Monte Carlo simulations.
+    MC_electrons  : "" 
+    MC_helium     : "" 
+    MC_protons    : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/Protons/dec_2276/sim_telarray" 
+    MC_gammadiff  : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/GammaDiffuse/dec_2276/sim_telarray/"
+    
+data_selection:
+    source_name_database: "CrabNebula"  # MUST BE THE SAME AS IN THE DATABASE; Set to null to process all sources in the given time range.
+    source_name_output: 'Crabtest'  # Name tag of your target. Used only if source_name_database != null.
```

Comment:
taking into account that we want to use those scripts to process all the data, I am not sure if allowing to have different output name is a good idea, do you have any use case for this in mind? If it is just for tests I think a better solution is to have a general directory prefix where you can specify where the files will land.

Reply:
The output name can and MUST be used only in case you process one source: e.g., a user wants to process some Crab data on its own; he can use different output names for different periods or different analyses (Crab_dec_2023, Crab_jan_2024....). Same for the input name (the one in the database): used in single-user analyses, not in massive data processing